config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java

```diff
@@ -173,6 +174,8 @@ private void addClusterContent(ContainerCluster cluster, Element spec, ConfigMod
         addLegacyFilters(spec, cluster);  // TODO: Remove for Vespa 7
 
         addAthensCopperArgos(cluster, context);  // Must be added after nodes.
+        cluster.setRpcServerEnabled(rpcServerEnabled);
```

Comment:
These set statements don't really belong in addClusterContent. Move up to doBuild() or create a new method.